src/main/java/raisetech/StudentManagement/StudentRepository.java

```diff
@@ -17,5 +19,6 @@ public interface StudentRepository {
     @Delete("DELETE FROM student WHERE name = #{name}")
     void deleteStudent(String name);
 
-
+    @Select("SELECT*FROM student")
```

Comment:
動くとは思いますが...
以下の方が良いですね。
```suggestion
    @Select("SELECT * FROM student")
```